anevar/nevar_DZ_8/DZ_8_3_1.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>DZ_8_3_2</title>
+	<style type="text/css">
+		.zebra {
+			border-color: red;
+			background-color: grey;
+		}
+		
+		.not {
+		    border-color: green;
+			background-color: lichtgreen;
+		}
+	</style>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+	<script type="text/javascript" src="https://code.jquery.com/jquery-3.3.1.js"></script>
+	<script type="text/javascript">
+	$(document).ready(function(){
+		$('#Password1').on("input", function(){
```

Comment:
To lot of lines.
You can use selector like this:
 $('#Password1, #Password2').....